lib/BubbleChart.js

```diff
@@ -13,13 +13,13 @@ const iface = {
     ...ChartBase.propTypes,
 
     data: PropTypes.shape({
-      datasets: PropTypes.arrayOf(PropTypes.shape({
+      dataSets: PropTypes.arrayOf(PropTypes.shape({
         values: PropTypes.arrayOf(PropTypes.shape({
           x: PropTypes.number,
           y: PropTypes.number.isRequired,
           size: PropTypes.number.isRequired,
+          marker:PropTypes.string,
```

Comment:
just like lineChart, if you add marker attribute here, then the content will displayed in marker when you select the entry.